#! /bin/sh /usr/share/dpatch/dpatch-run
## browser.dpatch by Samuel Mimram <smimram@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Use the default Debian browser for help.

@DPATCH@
diff -urNad coq-8.1.pl2+dfsg~/lib/options.ml coq-8.1.pl2+dfsg/lib/options.ml
--- coq-8.1.pl2+dfsg~/lib/options.ml	2007-10-15 11:00:52.000000000 +0000
+++ coq-8.1.pl2+dfsg/lib/options.ml	2007-10-15 11:01:51.000000000 +0000
@@ -118,7 +118,4 @@
            "\" must contain exactly one placeholder \"%s\".")
    else pre,post
  with
-  Not_found ->
-   if Sys.os_type = "Win32"
-   then "C:\\PROGRA~1\\INTERN~1\\IEXPLORE ", ""
-   else "firefox -remote \"OpenURL(", ")\""
+  Not_found -> "/usr/bin/x-www-browser ", ""
